common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -113,13 +113,13 @@ public SharedPreferencesFileManager(
             final String name,
             final IKeyAccessor encryptionManager) {
         if (encryptionManager == null) {
-            Logger.verbose(TAG, "Init: ");
+            Logger.verbose(TAG+"::constructor", "Init: ");
```

Comment:
nit: Let's use a string to make it consistent.

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8